lib/api_me/sorting.rb

```diff
@@ -52,5 +59,9 @@ def sorting?
       sort_criteria || sort_reverse
     end
 
+    def relation_sorting(str)
+      str.slice!(/_id/)
+      return str
+    end
```

Comment:
This was ended up working to fix a problem I was having joining the associated tables. For example, retailer_id needs to be sent across the server, but the word 'retailer' is needed to create the model constant for the join.

![image](https://cloud.githubusercontent.com/assets/13594010/20939742/cfa53800-bba4-11e6-8539-28b833842e5a.png)